src/common/expandable/Expandable.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+import Icon from '../icon/Icon';
+import styles from './expandable.module.scss';
+import Button from '../button/Button';
+
+interface Props {
+  label: React.ReactNode;
+  onClick?: Function;
+}
+
+const Expandable: React.SFC<Props> = ({ label, onClick, children }) => {
+  const [expanded, setExpanded] = useState(false);
+  const handleClick = () => {
+    setExpanded(!expanded);
+    onClick && onClick();
+  };
+
+  return (
+    <>
+      <div className={styles.expandable}>
+        <div className={classNames(styles.headerBtn, styles.label)}>
+          <Button variant="text" color="secondary" onClick={handleClick}>
+            {label}
+          </Button>
+        </div>
+        {children && (
+          <div
+            className={classNames(styles.headerBtn, styles.arrow, {
```

Comment:
![Screenshot from 2019-11-13 13-05-38](https://user-images.githubusercontent.com/3033037/68758354-58f94600-0616-11ea-9355-2882b29c4372.png)

Arrow should be pointing down when unexpanded and up when expanded.